packages/client/src/helpers/buildPaginatedQueryResult.ts

```diff
@@ -10,12 +8,10 @@ export type PaginatedQueryData<Item> = {
   items: Item[];
 };
 
-export type PaginatedResult<T> = Prettify<
-  PaginatedQueryData<T> & {
-    next(): Promise<PaginatedResult<T> | null>;
-    prev(): Promise<PaginatedResult<T> | null>;
-  }
->;
+export type PaginatedResult<T> = PaginatedQueryData<T> & {
```

Comment:
Does the `&` achieve the same result as `Prettify` in the resulting typedef in VS Code suggestions?